installation-bundle/src/InstallTool.php

```diff
@@ -206,6 +206,11 @@ public function hasConfigurationError(array &$context): bool
 
                 return true;
             }
+
+            // If there is no collation configured, use the default one
```

Comment:
Should this not be in an `} else {` block after `if (isset($options['collate'])) {`?